plugin/api.go

```diff
@@ -289,6 +289,15 @@ type API interface {
 	// UpdateChannelMemberNotifications updates a user's notification properties for a channel.
 	UpdateChannelMemberNotifications(channelId, userId string, notifications map[string]string) (*model.ChannelMember, *model.AppError)
 
+	// GetGroup gets a group by ID.
```

Comment:
should these contain `Minimum server version: 5.16` comment?

Reply:
Also what happens if these are invoked on team-edition and not EE-dition? Do they get an empty result back or an error that it's not implemented?

Reply:
Yes good catch, I'll add the comments.

If they are used by TE installation then they'll just return an error, similar to other plugin APIs that are for EE features like LDAP.

Reply:
Added for 5.18 as this missed the 5.16 deadline